homeassistant/components/hassio/websocket_api.py

```diff
@@ -111,11 +111,19 @@ async def websocket_supervisor_api(
         raise Unauthorized()
     supervisor: HassIO = hass.data[DOMAIN]
     try:
+        command = msg[ATTR_ENDPOINT]
+        payload = msg.get(ATTR_DATA, {})
+
+        if command == "/ingress/session":
+            # Send username on session creation, so the supervisor can correlate session tokens with users
+            # for every request that is authenticated with the given ingress session token.
+            payload["username"] = connection.user.name
+
```

Comment:
Don't think this needs to be inside the try. We keep try/exception block short as possible.